selene/src/main.rs

```diff
@@ -412,17 +461,8 @@ fn start(mut options: opts::Options) {
 
                 (config_contents, Path::new("-"))
             } else {
-                let config_path = Path::new("selene.toml");
-
-                let config_contents = match fs::read_to_string(config_path) {
-                    Ok(contents) => contents,
-                    Err(error) => {
-                        error!("Error reading config file: {error}");
-                        std::process::exit(1);
-                    }
-                };
-
-                (config_contents, config_path)
+                let (config_content, config_path) = parse_config_file_as_string().unwrap();

```

Comment:
Unwrap can panic. Need an error message here.

Reply:
I cannot repro the scenario in which this provoke panic.  All scenarios I've tried are successful:

* `.selene.toml` exists, and it is used.
*  `selene.toml` exists, and it is used.
* `config file` doesn't exist (defaults are loaded correctly).
* config file exists but some option is formatted incorrectly, so a notification is sent to the user, and exit (for `.selene.toml` and `selene.toml`).
* config exists but it's empty, so defaults are loaded (for `.selene.toml` and `selene.toml`).

Reply:
fixed in https://github.com/Kampfkarren/selene/pull/591/commits/ade463c78eca904e2d3fd801c3f6a792f258edc6